README.md

```diff
@@ -462,3 +468,24 @@ NOTE: when prompted, enter the password which is "upthehill".
 
 You can type a line of text and when you press enter, the line will be echoed
 back. Use CTRL-C to terminate the connection.
+
+
+CERTIFICATE SUPPORT
+===================
+
+wolfSSH can accept X.509 certificates in place of just public keys when
+authenticating a user. This feature is currently a work in process.
+
+To compile wolfSSH with X.509 support, use the `--enable-certs` build option
+or define `WOLFSSH_CERTS`:
+
+    $ ./configure --enable-certs
+    $ make
+
+To provide a CA root certificate to validate a user's certificate, give the
+echoserver the command line option `-a`.
+
+    $ ./examples/echoserver/echoserver -a ./keys/ca-cert.pem
```

Comment:
```% ./examples/echoserver/echoserver -a ./keys/ca-cert.pem
Couldn't find size of file ./keys/ca-cert.pem.```
Seems file is missing or wrong file is used here? Is it supposed to be keys/ca-ecc-cert.pem?